plugins/modules/zos_script.py

```diff
@@ -154,12 +148,6 @@
     remote_src: true
     chdir: /u/user/output_dir
 
-- name: Run a local Python script that uses a custom tmp_path.
```

Comment:
i think it's worth leaving a version of this example in the docs. i see you have a statement about how to set temp path in the `notes` section, but i think more users will jump straight to the `examples` section and possibly miss learning about the available config. I'm thinking the example could be something like:
```yml
- name: Run a local Python script in the temporary directory specified in the Ansible environment variable 'remote_tmp'.
   cmd: ./scripts/program.py
   executable: /usr/bin/python3
```
where just the playbook task name briefly highlights where the config is located/managed from.